packages/spec/integration/jest-preset/__tests__/preset.js

```diff
@@ -0,0 +1,6 @@
+describe('jest-preset', () => {
+  it('allows to use typescript', () => {
+    const calculator = require('./setup/calculator.ts').default;
```

Comment:
Do you need to use `.ts` here? Or could you just require `'./setup/calculator` instead?